tests/outputs/a_todo/people.yaml

```diff
@@ -0,0 +1,526 @@
+        amount_learned: 4.57
+        appropriateness_workload: 4.29
+        fairness_grading: 4.57
+        course_overall: 4.43
+        organization: 4.0
+        classroom_delivery: 4.29
+        approachability: 4.86
+        instructor_overall: 4.43
+        comments:
+          - Great teacher but disorganized
+          - Wears pink pants.  Why?
+      month: Jan
+      organization: Columbia University
+      position: professor
+      semester: Spring
+      syllabus: https://drive.google.com/open?id=0BxUpd34yizZreDBCMEJNY2FUbnc
+      year: 2018
+    - course: 'MSAE-3010: Introduction to Materials Science'
+      courseid: s17-3010
+      description: This course is an introduction to nuclear physics.
+      enrollment: 18
+      evaluation:
+        response_rate: 58.33
+        amount_learned: 4.57
+        appropriateness_workload: 4.29
+        fairness_grading: 4.57
+        course_overall: 4.43
+        organization: 4.0
+        classroom_delivery: 4.29
+        approachability: 4.86
+        instructor_overall: 4.43
+        comments:
+          - Great teacher but disorganized
+          - Wears pink pants.  Why?
+      month: Jan
+      organization: Columbia University
+      position: professor
+      semester: Spring
+      syllabus: https://drive.google.com/open?id=0BxUpd34yizZreDBCMEJNY2FUbnc
+      year: 2017
+    - course: 'MSAE-3010: Introduction to Materials Science'
+      courseid: s17-3010
+      description: This course is an introduction to nuclear physics.
+      enrollment: 18
+      month: Jan
+      organization: Columbia University
+      position: professor
+      semester: Spring
+      syllabus: https://drive.google.com/open?id=0BxUpd34yizZreDBCMEJNY2FUbnc
+      year: 2019
+    - course: 'MSAE-3010: Introduction to Materials Science'
+      courseid: f18-3010
+      description: This course is an introduction to nuclear physics.
+      enrollment: 18
+      evaluation:
+        response_rate: 58.33
+        amount_learned: 4.57
+        appropriateness_workload: 4.29
+        fairness_grading: 4.57
+        course_overall: 4.43
+        organization: 4.0
+        classroom_delivery: 4.29
+        approachability: 4.86
+        instructor_overall: 4.43
+        comments:
+          - Great teacher but disorganized
+          - Wears pink pants.  Why?
+      month: August
+      organization: Columbia University
+      position: professor
+      semester: Fall
+      syllabus: https://drive.google.com/open?id=0BxUpd34yizZreDBCMEJNY2FUbnc
+      year: 2018
+    - course: 'MSAE-3010: Introduction to Materials Science'
+      courseid: f19-3010
+      description: This course is an introduction to nuclear physics.
+      month: August
+      organization: Columbia University
+      position: professor
+      semester: Fall
+      syllabus: https://drive.google.com/open?id=0BxUpd34yizZreDBCMEJNY2FUbnc
+      year: 2019
+  title: Dr.
+  todos:
+    - description: read paper
+      due_date: '2020-07-19'
```

Comment:
I keep this as string type, because the first two tasks are from schemas. In the exemplars of schemas, it seems that the date can only be written with quotation mark, or it won't pass the test.